src/mono/wasm/debugger/BrowserDebugProxy/MonoProxy.cs

```diff
@@ -20,7 +20,7 @@ internal class MonoProxy : DevToolsProxy
         HashSet<SessionId> sessions = new HashSet<SessionId>();
         Dictionary<SessionId, ExecutionContext> contexts = new Dictionary<SessionId, ExecutionContext>();
 
-        public MonoProxy(ILoggerFactory loggerFactory, bool hideWebDriver = true) : base(loggerFactory) { hideWebDriver = true; }
+        public MonoProxy(ILoggerFactory loggerFactory, bool hideWebDriver = true) : base(loggerFactory) { this.hideWebDriver = true; }
```

Comment:
the changes here were necessary because the compiler failed due to warnings, and this one looks like an actual bug (we were assigning to the local instead of the class field)